fusor-ember-cli/app/templates/deployments.hbs

```diff
@@ -19,7 +19,9 @@
    </div>
 </div>
 </form>
-<br />
+<p class='displaying-entries'>
+  {{if isFiltered 'Found' 'Displaying'}} <strong>{{filteredDeployments.length}} of {{totalDeployments}}</strong> entries
+</p>
```

Comment:
Adding "Displaying 20 of 57" above and below. If it's below, then the user doesn't see the total until scrolling down.